ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -9,22 +9,26 @@
  */
 package org.openmrs.reference;
 
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.page.HomePage;
 
 import static org.junit.Assert.assertTrue;
 
-@Ignore
+
 public class LoginTest extends ReferenceApplicationTestBase {
     private HomePage homePage;
+    
+    public void initiateHomePage() {
```

Comment:
any reasons for making this public?

Reply:
This allows any other class to be visible to all other classes

Reply:
When a method is consumed by functions within the same class, making it private is the best alternative. On the other hand  when a method is invoked by functions outside the host class then making it public or protected overrides private modifier. I find this [article](https://stackoverflow.com/questions/215497/what-is-the-difference-between-public-protected-package-private-and-private-in) helpful.

Reply:
/This method should be generic , Making private would make it not  accessible /.reusable. am assuming that we can as well call this method even out side this calss cc @k-joseph  @kdaud  , i think we might need it to be  public such it can easily be called even out side this class, your suggestions will be appreciated